src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Endpoint.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Extensions.Logging;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring.Validation.Test
```

Comment:
please use shorter namespaces. NuGet.Services.Metadata.Catalog. should be sufficient here. Having long namespaces just makes you use excessive using statements.